lib/SourceMapDevToolPlugin.js

```diff
@@ -236,6 +304,9 @@ class SourceMapDevToolPlugin {
 						assetsCache.set(asset, { file, assets });
 						/** @type {string | false} */
 						let currentSourceMappingURLComment = sourceMappingURLComment;
+						/**
+						 * In css comment with `//` commenting next block as well, so workaround is to replace comment `//` with `/**\/`
```

Comment:
In CSS there are only `/*  */` comments, `//` comments doesn't exist.

Reply:
my fault, i was reading about *suggested* syntax